kubelet-to-gcm/README.md

```diff
@@ -0,0 +1,85 @@
+# addon-resizer
```

Comment:
I must be missing something. This file is about addon-resize, while the rest of the code looks unrelated. Is it a mistake?